eng/Tools.props

```diff
@@ -37,21 +42,28 @@
     <PropertyGroup>
       <ToolsDir>$(RepoRoot)Tools</ToolsDir>
       <BuildToolsSemaphore>$(ToolsDir)/$(BuildToolsPackageVersion).init-tools.completed</BuildToolsSemaphore>
-      <BuildToolsPackageDir>$(NuGetPackageRoot)/$(BuildToolsPackage)/$(BuildToolsPackageVersion)/lib/</BuildToolsPackageDir>
+      <PackagesDir>$(NuGetPackageRoot)/</PackagesDir>
+      <BuildToolsPackageDir>$(PackagesDir)$(BuildToolsPackage)/$(BuildToolsPackageVersion)/lib/</BuildToolsPackageDir>
       <CmdExt>cmd</CmdExt>
       <CmdExt Condition="'$(OS)' != 'Windows_NT'">sh</CmdExt>
       <DotNetRoot Condition="'$(DotNetRoot)' == ''">$(RepoRoot).dotnet\</DotNetRoot>
       <DotNetCmd>$(DotNetRoot)dotnet</DotNetCmd>
       <DotNetCmd Condition="'$(OS)' == 'Windows_NT'">$(DotNetCmd).exe</DotNetCmd>
 
-      <InitToolsCmdLine>$(BuildToolsPackageDir)init-tools.$(CmdExt) $(RepoRoot) $(DotNETCmd) $(ToolsDir) $(NuGetPackageRoot)\</InitToolsCmdLine>
+      <InitToolsCmdLine>$(BuildToolsPackageDir)init-tools.$(CmdExt) $(RepoRoot) $(DotNETCmd) $(ToolsDir) $(PackagesDir)</InitToolsCmdLine>
     </PropertyGroup>
 
     <Exec
         Condition="!Exists('$(BuildToolsSemaphore)')"
         Command="$(InitToolsCmdLine) &gt; $(RepoRoot)init-tools.log" />
 
     <Touch Files="$(BuildToolsSemaphore)" AlwaysCreate="true" />
+    <OnError ExecuteTargets="DumpInitToolsLog" />
+  </Target>
+
+  <Target Name="DumpInitToolsLog">
+    <Message Text="Init-tools failed dumping log file $(RepoRoot)init-tools.log" />
+    <Exec Condition="Exists('$(RepoRoot)init-tools.log')" Command="type $(RepoRoot)init-tools.log" />
```

Comment:
Does this work on Unix?  Maybe `<Message Text="$([System.IO.File]::ReadAllText('$(RepoRoot)init-tools.log'))" />`

Reply:
Good catch I'll update this.